diff -r -u wireshark-0.99.8/configure.in wireshark-0.99.8-ca/configure.in
--- wireshark-0.99.8/configure.in	2008-02-27 20:59:35.000000000 +0100
+++ wireshark-0.99.8-ca/configure.in	2008-03-13 16:12:28.000000000 +0100
@@ -1678,6 +1678,7 @@
   plugins/agentx/Makefile
   plugins/artnet/Makefile
   plugins/asn1/Makefile
+  plugins/ca/Makefile
   plugins/ciscosm/Makefile
   plugins/docsis/Makefile
   plugins/enttec/Makefile
diff -r -u wireshark-0.99.8/Makefile.am wireshark-0.99.8-ca/Makefile.am
--- wireshark-0.99.8/Makefile.am	2008-02-27 20:59:35.000000000 +0100
+++ wireshark-0.99.8-ca/Makefile.am	2008-03-13 16:12:28.000000000 +0100
@@ -239,6 +239,7 @@
 	-dlopen plugins/agentx/agentx.la \
 	-dlopen plugins/artnet/artnet.la \
 	-dlopen plugins/asn1/asn1.la \
+	-dlopen plugins/ca/ca.la \
 	-dlopen plugins/ciscosm/ciscosm.la \
 	-dlopen plugins/docsis/docsis.la \
 	-dlopen plugins/enttec/enttec.la \
diff -r -u wireshark-0.99.8/plugins/Makefile.am wireshark-0.99.8-ca/plugins/Makefile.am
--- wireshark-0.99.8/plugins/Makefile.am	2008-02-27 20:49:42.000000000 +0100
+++ wireshark-0.99.8-ca/plugins/Makefile.am	2008-03-13 16:12:28.000000000 +0100
@@ -25,6 +25,7 @@
 	agentx \
 	artnet \
 	asn1 \
+	ca \
 	ciscosm \
 	docsis \
 	enttec \
diff -r -u wireshark-0.99.8/plugins/Makefile.in wireshark-0.99.8-ca/plugins/Makefile.in
--- wireshark-0.99.8/plugins/Makefile.in	2008-02-27 20:59:50.000000000 +0100
+++ wireshark-0.99.8-ca/plugins/Makefile.in	2008-03-13 16:12:28.000000000 +0100
@@ -320,6 +320,7 @@
 	agentx \
 	artnet \
 	asn1 \
+	ca \
 	ciscosm \
 	docsis \
 	enttec \
diff -r -u wireshark-0.99.8/plugins/Makefile.nmake wireshark-0.99.8-ca/plugins/Makefile.nmake
--- wireshark-0.99.8/plugins/Makefile.nmake	2008-02-27 20:49:42.000000000 +0100
+++ wireshark-0.99.8-ca/plugins/Makefile.nmake	2008-03-13 16:18:01.000000000 +0100
@@ -38,6 +38,9 @@
 	cd asn1
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
+	cd ca
+	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
+	cd ..
 	cd ciscosm
 	$(MAKE) /$(MAKEFLAGS) -f Makefile.nmake $(PLUGIN_TARGET)
 	cd ..
@@ -121,6 +124,7 @@
 	xcopy agentx\*.dll $(VERSION) /d
 	xcopy artnet\*.dll $(VERSION) /d
 	xcopy asn1\*.dll $(VERSION) /d
+	xcopy ca\*.dll $(VERSION) /d
 	xcopy ciscosm\*.dll $(VERSION) /d
 	xcopy docsis\*.dll $(VERSION) /d
 	xcopy enttec\*.dll $(VERSION) /d
